cli/src/main/java/com/box/l10n/mojito/cli/command/ThirdPartySyncCommand.java

```diff
@@ -38,6 +43,12 @@
     @Parameter(names = {"--project", "-p"}, arity = 1, required = true, description = "Third party project to synchronize with")
     String thirdPartyProjectId;
 
+    @Parameter(names = {"--actions", "-a"}, arity = 1, required = false, description = "Actions to synchronize", listConverter = ThirdPartySyncActionsConverter.class)
```

Comment:
i think it should be variableArity = true, since it is a list. Or do you want to convert a single param into the list